### PR TITLE
General Improvements for Documentation

### PR DESCRIPTION
**Pull request by TheCedarPrince**
These are general improvements for documentation based on #114 

Let me know what you think and I can go ahead and merge these!

### CODE REVIEW

File: docs/src/index.md [hunk 1]
```diff
@@ -31,4 +31,11 @@ npm install -g mathjax-node-cli
 ```
 
 As of this moment, we do not have native `LaTeX` rendering in Julia.
-We look forward to the day when we can remove this from our documentation! 
+We look forward to the day when we can remove this from our documentation!
+
+What is also required is the great package, `LaTeXStrings`.
```
Comment (Wikunia) [inline docs/src/index.md:36]: Maybe a link to the package?

File: docs/src/tutorials/tutorial_3.md [hunk 2]
```diff
@@ -72,6 +72,12 @@ Currently, Julia does not have the ability (yet) to render `LaTeX` natively.
 Therefore, we must install an additional node package.
 Sadly. 😭
 
+Furthermore, we do need to install an additional Julia package called `LaTeXStrings`.
```
Comment (Wikunia) [inline docs/src/tutorials/tutorial_3.md:75]: And here as well 😊

File: test.jl [hunk 3]
```diff
@@ -0,0 +1,24 @@
+using Javis
+using LaTeXStrings
```
Comment (Wikunia) [inline test.jl:2]: This time should be in git ignore I think.

File: docs/src/tutorials/tutorial_4.md [hunk 4]
```diff
@@ -26,6 +26,7 @@ From this tutorial, you will learn how to:
 ## Starting with the Basics of SubAction
 
 The `ground` function should be familiar to you as well as the general structure of the `javis` function if you have seen the first [tutorial](tutorial_1.md).
+In this tutorial, rather than calling the `javis` function by itself, we are going to be calling it from the function we are creating to create our mascot, `face`:
```
Comment (Wikunia) [inline docs/src/tutorials/tutorial_4.md:29]: Maybe call `face()` in the final code snippet such that copy pasting it in the REPL will actually work.

**Review by Wikunia — COMMENTED** [review]
Thanks! Just some minor comments to hopefully make it more accessible

**Review by Wikunia — APPROVED** [review]
LGTM. Feel free to merge if you have done everything you wanted. Sorry for the later review :smile:

**Comment by TheCedarPrince** [comment]
Haha, no problem!

**Pull request merged**